### ISSUE TITLE
Can't change Padding of Row for Responsive mode

### ISSUE THREAD

**Issue by menachemshapiro** [livecomposer/live-composer-page-builder#194]
Website is here: http://emilyatal.com

TL;DR: Can't change vertical (or horizontal) padding of rows in responsive mode.

-----------------
I have vertical padding of each row set to 80. I'd like to make it less in Responsive mode (Phone view), but there is no responsive option for the rows, only the modules.

I found a workaround for some of the rows (e.g. the first one). I was able to set the "Margin Top" of the navigation module to -56, which did the same thing as reducing the vertical padding. 

Other rows, where the image module was the top module, this was not an option because there is no "Margin Top" option for the image module.

**Comment by livecomposer** [comment]
Hi there,
Unfortunately this is how ROW system created. We are planing to completely recode ROWs so they work as modules. For now please create a custom class for responsive design and apply it to the rows.

For example:

    @media only screen and ( max-width: 767px ) {
        .no-responsive-padding {
            padding-top:0;
            padding-bottom:0;
        }
    }

Then you can add **no-responsive-padding** to any row that should have no padding on mobile.

For tablets use;

    @media only screen and (min-width : 768px) and (max-width : 1024px)  {
    ...
    }

I hope it helps.
